contracts/test/benefactorLeave.js

```diff
@@ -0,0 +1,200 @@
+const {
+  BN,
+  expectRevert,
+  ether,
+  expectEvent,
+  balance,
+  time,
+} = require('@openzeppelin/test-helpers');
+
+const NoLossDao = artifacts.require('NoLossDao');
+const AaveLendingPool = artifacts.require('AaveLendingPool');
+const ERC20token = artifacts.require('MockERC20');
+const ADai = artifacts.require('ADai');
+
+contract('NoLossDao', accounts => {
+  let aaveLendingPool;
+  let noLossDao;
+  let dai;
+  let aDai;
+
+  const applicationAmount = '5000000';
+
+  beforeEach(async () => {
```

Comment:
We can refactor this before each to call a common setup function rather than copy pasting everywhere.